package.json

```diff
@@ -9,7 +9,6 @@
   "module": "dist/webmain.js",
   "main": "dist/index.es.js",
   "scripts": {
-    "prepare": "cross-env tools/sigh check && cd devtools && npm i && cd ../concrete-storage && npm i",
```

Comment:
you could work around this by having a separate package.json for Bazel I guess

Reply:
I tried this; it didn't work. Somewhere the rules_nodejs package looks for a file called `package.json` in the root directory, despite the argument that one supplies for the package file. 

I did find out what the problem was and filed a bug for it #5379.